pkg/controller/admin/users_list.go

```diff
@@ -56,6 +56,30 @@ func (c *Controller) HandleUsersIndex() http.Handler {
 	})
 }
 
+// HandleUserShow renders details about a user.
+func (c *Controller) HandleUserShow() http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		ctx := r.Context()
+		vars := mux.Vars(r)
+
+		// Pull the user from the id.
+		user, err := c.db.FindUser(vars["id"])
+		if err != nil {
+			if database.IsNotFound(err) {
+				controller.Unauthorized(w, r, c.h)
```

Comment:
This _probably_ doesn't make as much sense here, since this is guarded by the system admin. We should probably return a not-found or something.

Reply:
agreed